package.json

```diff
@@ -20,7 +20,10 @@
     "javascript"
   ],
   "devDependencies": {
-    "lerna": "^2.5.1"
+    "husky": "1.0.0-rc.12",
```

Comment:
The good part with `husky@next` is that it knows when you commit a partially staged file. 

Used with `pretty-quick`, if the file is modified during precommit hook (because Prettier), then it'll warn you before adding the whole file again. 

So you can partially stage files without having precommit hook adding the whole stuff on fixes 🎉 

![image](https://user-images.githubusercontent.com/1094774/42412677-e2f81178-81de-11e8-86b2-ca7f0f3a9bbe.png)

Reply:
My hero! :tada: This look very amazing!